### ISSUE TITLE
Add a data source that IntelliJ can understand

### ISSUE THREAD

**Issue by jonbarksdale** [wolpert/dstore#18]
Should help us catch any SQL bugs, as IntellJ knows that JDBI statements are SQL, and automatically tries to validate them with a data source. 

We can commit datasources as part of the project for discoverability.

There are a few options I can see: 

1. Add a task to the build that will generate an HSQLDB file(s) from the liquibase scripts, which can then be used as a datasource
2. convert the Liquibase DDL to DDL files, so intelliJ can use the directory as a DDL DataSource (assuming this is possible)
3. Convert from Liquibase to Flyway or something that works off of DDL scripts, and use those scripts as a DDL data source. 
4. Add a build task for Liquibase to generate DDL scripts (update-sql command?)

**Comment by wolpert** [comment]
I feel like there should be a liquibase/intelij plugin.

**Comment by wolpert** [comment]
Nope... or at least, not what we want it to do. (And out of date) https://plugins.jetbrains.com/plugin/11510-liquibase-changesets-generator
